### DISCUSSION TITLE
QGIS environment- How to stop it?

### DISCUSSION THREAD

**Discussion by ddjustina** [microsoft/PlanetaryComputer#148] [Q&A] [answered]
I was using the QGIS environment, but I need to switch back to jupyter hub, but I can turnoff the QGIS environment.

**Comment by TomAugspurger** [comment, marked as answer]
Can you try going to https://pccompute.westeurope.cloudapp.azure.com/compute/hub/home? That should have what you need to stop / start your servers.

**Reply by ddjustina** [reply]
Thanks @TomAugspurger !
